### PR TITLE
{Term Entry] Python:NumPy Math Methods power

### PR DESCRIPTION
**Pull request by cigar-galaxy82**
### Description

Added new term entry for Python:NumPy Math Methods `power`

### Issue Solved

Closes [#4711](https://github.com/Codecademy/docs/issues/4711)

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/math-methods/terms/power/power.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+## Syntax
+
+```pseudo
+numpy.round(array1, array2, out=None, where=True)
+```
+
+- `array1`: A number or an array of numbers which will be used as a base.
+- `array2`: A number or an array of numbers which will be used as exponent for elements in `array1`.
+- `out`: An optional parameter that allows us to store the output array. If not provided, a new array will be created.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/power/power.md:28]:
```suggestion
- `out`: An optional parameter that allows us to store the output array. If not specified, a new array will be allocated for the result.
```

Comment (mamtawardhani) [reply content/numpy/concepts/math-methods/terms/power/power.md:28]: reframed it due to plagiarism issues for the second half of the sentence

File: content/numpy/concepts/math-methods/terms/power/power.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+```shell
+2^3 is equal to 8
+
+Output array : [64 243 256  25  36]
+```
+
+## Codebyte Example
+
+In this codebyte example, the `.power()` computes the power of the given elements in the in the `array1` as base with elements in `array2` as exponent for corresnpoding elements in `array1`:
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/power/power.md:59]:
```suggestion
In this codebyte example, `.power()` method computes the power where each element in `array1` serves as the base raised to the power of the corresponding element in `array2`:
```

Comment (mamtawardhani) [reply content/numpy/concepts/math-methods/terms/power/power.md:59]: corrected the spelling mistakes and sentence formation

File: content/numpy/concepts/math-methods/terms/power/power.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+In the NumPy library, the **`.power()`** method raises each element in the first array to the corresponding element in the second array and then calculates it. It returns an array without commas separating the elements. To view the output with commas, use [`.repr()`](https://www.codecademy.com/resources/docs/python/built-in-functions/repr).
+
+## Syntax
+
+```pseudo
+numpy.round(array1, array2, out=None, where=True)
```
[the remark this answers is not in the document]
Comment (mamtawardhani) [reply content/numpy/concepts/math-methods/terms/power/power.md:23]: 1. the entry is for the `numpy.power()` method, the syntax was having `.round()`
2. Let's stick to the official NumPy doc for the parameters: https://numpy.org/doc/stable/reference/generated/numpy.power.html

File: content/numpy/concepts/math-methods/terms/power/power.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+- `array1`: A number or an array of numbers which will be used as a base.
+- `array2`: A number or an array of numbers which will be used as exponent for elements in `array1`.
```
Comment (mamtawardhani) [inline content/numpy/concepts/math-methods/terms/power/power.md:27]:
```suggestion
- `x1`: A number or an array of numbers used as the base.
- `x2`: A number or an array of numbers used as the exponent for elements in `x1`.
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Thank you for contributing to Docs @cigar-galaxy82, the entry was well written! 😃 

I've suggested a few changes, could you please review and modify those at your earliest convenience? Feel free to discuss any concerns, thank you! 😄

**Comment by mamtawardhani** [comment]
@cigar-galaxy82 please make the required changes, so this can be merged. Thank you 😃

**Comment by cigar-galaxy82** [comment]
Hi @mamtawardhani all the recommended changes are added

**Review by mamtawardhani — APPROVED** [review]
Great entry @cigar-galaxy82 😃 

Ready to be merged! 🚀

**Pull request merged**